docker/fedora-36.dockerfile

```diff
@@ -22,6 +22,7 @@ dnf -y install \
   cmake-3.22.2* \
   gcc-12.0.1* \
   gcc-c++-12.0.1* \
+  intel-mediasdk-devel-22.3.0* \
```

Comment:
Can you put this in the removed if statement? Only, needed for amd64.

Applies to both images.